.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Actual Behavior
```

Comment:
Please flip these. Usually you raise an issue/PR because the expected is not happening. So it's more natural to say "expected X, but got Y".